mne/io/eeglab/eeglab.py

```diff
@@ -0,0 +1,360 @@
+        If True, the data will be preloaded into memory (fast, requires
+        large amount of memory). If preload is a string, preload is the
+        file name of a memory-mapped file which is used to store the data
+        on the hard drive (slower, requires less memory). Note that
+        preload=False will be effective only if the data is stored in a
+        separate binary file.
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+
+    Returns
+    -------
+    raw : Instance of RawEEGLAB
+        A Raw object containing EEGLAB .set data.
+
+    Notes
+    -----
+    .. versionadded:: 0.11.0
+
+    See Also
+    --------
+    mne.io.Raw : Documentation of attribute and methods.
+    """
+    return RawEEGLAB(input_fname=input_fname, montage=montage, eog=eog,
+                     preload=preload, verbose=verbose)
+
+
+def read_epochs_eeglab(input_fname, events=None, event_id=None, montage=None,
+                       verbose=None):
+    """Reader function for EEGLAB epochs files
+
+    Parameters
+    ----------
+    input_fname : str
+        Path to the .set file.
+    events : str | array, shape (n_events, 3) | None
+        Path to events file. If array, it is the events typically returned
+        by the read_events function. If some events don't match the events
+        of interest as specified by event_id,they will be marked as 'IGNORED'
+        in the drop log. If None, it is constructed from the EEGLAB (.set) file
+        with each unique event encoded with a different integer.
+    event_id : int | list of int | dict | None
+        The id of the event to consider. If dict,
+        the keys can later be used to acces associated events. Example:
+        dict(auditory=1, visual=3). If int, a dict will be created with
+        the id as string. If a list, all events with the IDs specified
+        in the list are used. If None, the event_id is constructed from the
+        EEGLAB (.set) file with each descriptions copied from `eventtype`.
+    montage : str | None | instance of montage
+        Path or instance of montage containing electrode positions.
+        If None, sensor locations are (0,0,0). See the documentation of
+        :func:`mne.channels.read_montage` for more information.
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+
+    Returns
+    -------
+    epochs : instance of Epochs
+        The epochs.
+
+    Notes
+    -----
+    .. versionadded:: 0.11.0
+
+
+    See Also
+    --------
+    mne.Epochs : Documentation of attribute and methods.
+    """
+    epochs = EpochsEEGLAB(input_fname=input_fname, events=events,
+                          event_id=event_id, montage=montage, verbose=verbose)
+    return epochs
+
+
+class RawEEGLAB(_BaseRaw):
+    """Raw object from EEGLAB .set file.
+
+    Parameters
+    ----------
+    input_fname : str
+        Path to the .set file.
```

Comment:
+1 for saying it in the docstring and throwing an infomative error if the file is not present